src/App.tsx

```diff
@@ -13,7 +14,18 @@ const StyledNav = styled.nav`
   min-width: 200px;
 `;
 
+const StyledButton = styled.button`
+  background-color: ${({ theme }) => theme.colors.blue};
+  color: ${({ theme }) => theme.colors.white};
+  border: none;
+  border-radius: 0.5rem;
+  text-align: center;
+  margin-left: 6rem;
```

Comment:
No need to fix necessarily - more of FYI:

It's often not a good idea to add margins to individual components to 'space it out' from other elements / borders etc because where & how far the elements are placed should be a contextual concern for the container of the component (hope I'm making sense - happy to go into details IRL if needed). It's a simple project and I myself could have contradicted myself a few times here and there 😛 but wanted to let you know for future information and your actual serious work.

Reply:
aww I had no idea so thanks for the explanation!